cosmos-gui/public/stylesheets/style.styl

```diff
@@ -70,6 +70,12 @@ header
     padding 0px 10px
     margin-bottom 10px
 
+  a
```

Comment:
Just curious, any reason to mix CSS files with Stylus ones??? :) I mean, is Stylus really needed? :)